### ISSUE TITLE
Categorical accuracy - ignore index

### ISSUE THREAD

**Issue by ethanwharris** [ecs-vlc/torchbearer#308]
See discussion from #306 would be nice to have an ignore_index option in the accuracy, like nn.CrossEntropyLoss

**Comment by ethanwharris** [comment]
@kl2792 This should now be added once #326 is merged. Use with the following:
`from torchbearer.metrics import CategoricalAccuracyFactory`
`metrics=[CategoricalAccuracyFactory(ignore_index=...)]`
